amplipi/streams.py

```diff
@@ -1167,6 +1200,14 @@ def send_cmd(self, cmd):
     except Exception:
       pass
 
+  def validate_stream(self, **kwargs):
+    url_regex = r'^https?://[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)$'
+    logo_regex = r'^https?://[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)\.(?:jpg|gif|png)$'
```

Comment:
We probably don't need to specify the type of image here. Web browsers support a wide variety of image formats and I assume that they don't require an extension.

Reply:
I assume both fields could just use url_regex right?